src/man/sssd.conf.5.xml

```diff
@@ -3779,6 +3779,25 @@ pam_gssapi_indicators_map = sudo:pkinit, sudo-i:pkinit
                     </listitem>
                 </varlistentry>
 
+                <varlistentry>
+                    <term>failover_primary_timeout (integer)</term>
```

Comment:
Is it worth update of `man sssd-ldap::FAILOVER` section?

Reply:
No, probably not. We didn't encounter issues with this setting being hardcoded before and even the customer confirmed that it is needed solely for their auditing tests when every server in the site is offline for a very long time, but has little to no value in real life scenario. We are implementing this only to satisfy their tests as a short term solution. Long term solution will be complete redesign of the failover.

Reply:
ok